### PR TITLE
Update data path.

### PR DESCRIPTION
**Pull request by junmeiban**
### Description
This PR is about updating the data path:
1. Change the path for "analysisDir" in config/jedi/ObsPlugs/retrieveObsErrors/trainAll.csh
2. Change the path for bias correction files and crtm_2.4.1 coefs in initialize/data/Observations.py
3. Change the path for 120km/60km/30km GFSANA in scenarios/defaults/externalanalyses.yaml
4. Change the path for 120km/60km/30km_1stCycle_background (in single precision) : in scenarios/defaults/firstbackground.yaml
5. The obs links (in scenarios/defaults/observations.yaml) are pointed to @liujake ’s pandac_common directory; I have not touched obs path in  scenarios/*.yaml yet.
6. Change the path for 60km_EnsFC_80EDA and 120km_GEFS_ensemble(20-member):  in scenarios/defaults/variational.yaml

### Files modified:
M       config/jedi/ObsPlugs/retrieveObsErrors/trainAll.csh
M       initialize/data/Observations.py
M       scenarios/defaults/externalanalyses.yaml
M       scenarios/defaults/firstbackground.yaml
M       scenarios/defaults/observations.yaml
M       scenarios/defaults/variational.yaml

P.S.:
The issue #225 cannot be closed so far.
The path for static B, ensemble localization, mpas_bundle(single precision), and mpas_model (outside bundle, intel+mpt)   will be updated in a separated PR. 

#### Tier 1:
 - [x] 3dhybrid_O30kmIE60km_SpecifiedEnsemble_VarBC.yaml

### CODE REVIEW

**Comment by junmeiban** [comment]
Reviewers can check my working directory if you are not available to run cycling experiments.
/glade/scratch/jban/pandac/jban_3dhybrid-60-60-iter_O30kmI60km_ensB-SE80+RTPP70_VarBC

File: scenarios/defaults/firstbackground.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ firstbackground:
         60km: # only available 20180414T18, 20200723T18
           directory: "/glade/p/mmm/parc/guerrett/pandac/fixed_input/60km/SingleFCFirstCycle/{{FirstCycleDate}}"
```
Comment (liujake) [inline scenarios/defaults/firstbackground.yaml]: Should replace 60km one as well. @byoung-joo can you confirm where is the 60km first cycle background you use? also 60km_GFSANA location. I can copy them over.

Comment (liujake) [reply scenarios/defaults/firstbackground.yaml]: @junmeiban 120km firstcycle background should also be changed in this PR.

File: scenarios/defaults/variational.yaml [hunk 2]
```diff
@@ -78,7 +78,7 @@ variational:
         EDA:
           60km:
             maxMembers: 80
-            directory0: '/glade/scratch/guerrett/pandac/guerrett_eda_3denvar-60-iter_NMEM80_RTPP0.7_SelfExclusion_OIE60km_WarmStart/CyclingFC/{{prevDateTime}}'
+            directory0: '/glade/p/mmm/parc/liuz/pandac_common/60km/60km_EnsFC_80EDA/{{prevDateTime}}'
```
Comment (liujake) [inline scenarios/defaults/variational.yaml:81]: 20-member 120km GEFS ensemble location should be updated in this PR.

**Comment by junmeiban** [comment]
Updated path for 60km first cycle background and 60km_GFSANA according to Jake's data structure. Waiting for Jake to copy data over.

File: scenarios/defaults/firstbackground.yaml (hunk 1 above)
Comment (byoung-joo) [reply scenarios/defaults/firstbackground.yaml]: For 60km first cycle background, I used the JJ's one `/glade/p/mmm/parc/guerrett/pandac/fixed_input/60km/SingleFCFirstCycle/2018041418`, which is linked from `/glade/p/mmm/parc/jban/pandac/60km_2018_1stCycle_background/2018041418`.

Comment (byoung-joo) [reply scenarios/defaults/firstbackground.yaml]: For 60 km GFSANA, I also used the JJ's one `/glade/p/mmm/parc/guerrett/pandac/fixed_input/60km/GFSAna/x1.163842.init.${datetime}.nc`, linked from `/glade/p/mmm/parc/guerrett/pandac/fixed_input/60km/GenerateGFSAnalyses/yyyymmddhh/x1.163842.init.${datetime}.nc`.

**Comment by liujake** [comment]
Now '60km_1stCycle_background_single' and '60km_GFSANA' are copied over to my ~60km folder.

File: initialize/data/Observations.py [hunk 3]
```diff
@@ -75,12 +75,12 @@ class Observations(Component):
     'GDASObsErrtable': ['/glade/work/guerrett/pandac/fixed_input/GSI_errtables/HRRRENS_errtable_10sep2018.r3dv', str],
 
     ## CRTM
-    'CRTMTABLES': ['/glade/work/guerrett/pandac/fixed_input/crtm_bin/', str],
+    'CRTMTABLES': ['/glade/p/mmm/parc/liuz/pandac_common/crtm_coeffs_v2.4.1', str],
```
Comment (liujake) [inline initialize/data/Observations.py]: need a '/' after crtm_coeffs_v2.4.1?

**Comment by junmeiban** [comment]
Also updated PR description. Now ready for review.

File: scenarios/defaults/variational.yaml (hunk 2 above)
Comment (liujake) [reply scenarios/defaults/variational.yaml:81]: 20-member 120km GEFS-based ensemble location needs to update to my location.

File: initialize/data/Observations.py (hunk 3 above)
Comment (junmeiban) [reply initialize/data/Observations.py]: Yes. need a "/".  Many thanks

File: scenarios/defaults/variational.yaml [hunk 4]
```diff
@@ -44,7 +44,7 @@ variational:
         GEFS:
           120km:
             maxMembers: 20
-            directory0: '/glade/p/mmm/parc/guerrett/pandac/fixed_input/120km/EnsForCov/{{prevDateTime}}'
+            directory0: '/glade/p/mmm/parc/liuz/pandac_common/120km/120km_EnsFC/{{prevDateTime}}'
             filePrefix: EnsForCov
```
Comment (junmeiban) [inline scenarios/defaults/variational.yaml:48]: Updated 20-member 120km GEFS-based ensemble location.

**Review by liujake — APPROVED** [review]

**Pull request merged**